code/modules/mob/living/life.dm

```diff
@@ -161,7 +161,7 @@
 
 /mob/living/proc/handle_disabilities()
 	//Eyes
-	if(disabilities & BLIND || stat)	//blindness from disability or unconsciousness doesn't get better on its own
+	if(sdisabilities & BLIND || stat)	//blindness from disability or unconsciousness doesn't get better on its own
```

Comment:
I really hate there's `sdisaiblities` and `disabilities` when just `disabilities` for all of them would be fine...